woocommerce-gutenberg-products-block.php

```diff
@@ -20,6 +20,7 @@
 
 $minimum_wp_version = '5.2';
 
+define( 'WC_BLOCKS_IS_FEATURE_PLUGIN', true );
```

Comment:
Maybe check if this is defined first just in case.

Reply:
Agreed! That would also allow for overriding in `wp-config.php` if necessary.